allennlp/data/dataset_readers/semantic_role_labeling.py

```diff
@@ -37,6 +86,10 @@ class SrlReader(DatasetReader):
     domain_identifier: ``str``, (default = None)
         A string denoting a sub-domain of the Ontonotes 5.0 dataset to use. If present, only
         conll files under paths containing this domain identifier will be processed.
+    bert_model_name : ``Optional[str]``, (default = None)
+        The BERT model to be wrapped, which we use to wordpiece-tokenize the data correctly.
+        If a bert model name is passed, we will also convert the BIO tags and verb indices
+        to be faithful to the wordpieces generated from the tokenizer.
```

Comment:
I feel like this comment should make it much more explicit that

* if you specify a `bert_model` here, then we will assume you want to use BERT throughout, will use the bert tokenizer, and will expand your tags and verb indicators accordingly
* if you don't specify a bert_model here, then it's just business as usual